Ramda/Extensions/drop.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  drop.swift
+//  Ramda
+//
+//  Created by Christian Leovido on 27/09/2019.
+//
+
+import Foundation
+
+extension R {
+
+    /**
+
+     Returns all but the first n elements
+     of the given list, string, or transducer/transformer (or object with a drop method).
+
+     Dispatches to the drop method of the second argument, if present.
+
+     - parameter iteration: Number of first elements to remove
+     - parameter string: The string to remove the values from
+
+     - returns: A new string with the first n elements removed
+
+     */
+    public class func drop(_ iteration: Int, _ string: String) -> String {
+
+        var newString = string
+
+       for _ in 0..<iteration {
+            if newString.count == 0 {
+                break
+            } else {
+                newString.removeFirst()
+            }
+        }
+
+        return newString
```

Comment:
We are missing the curried functions for both methods.

Reply:
I'm unsure about what you exactly mean with the curried functions. 

Following the Ramda.js documentation, I have changed the function to support a generic type T: Equatable

```
Number → [a] → [a]
Number → String → String
```

`public class func dropLast<T: Equatable>(_ iteration: Int, _ array: [T]) -> [T] {}`